apps/mark-scan/backend/src/custom-paper-handler/state_machine.ts

```diff
@@ -1172,6 +1156,27 @@ export function buildMachine(
             },
           },
         },
+        // This state is intentionally placed outside of the voting_flow block because we don't
+        // want to poll the paper handler status while we're resetting the paper handler. Doing so
+        // can result in unspecified erroneous behavior. The voting_flow block consistently polls
+        // paper handler status by way of polling the cover open status.
```

Comment:
See `Commit 2: No more hanging after jam clearing` in the PR description for more details